ai_agent/gemini_model.py

```
@@ -104,18 +104,22 @@ async def generate_ai_character_response(
 
-    os.makedirs("generated_images", exist_ok=True)
+        os.makedirs("generated_images", exist_ok=True)
 
-    filename = f"{uuid.uuid4()}.png"
-    file_path = os.path.join("generated_images", filename)
-    response.generated_images[0].image.save(file_path)
-    return file_path
+        filename = f"{uuid.uuid4()}.png"
+        file_path = os.path.join("generated_images", filename)
+        response.generated_images[0].image.save(file_path)
+        return file_path
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)